data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Northern Blockade"
+	minor
+	name `Mebsuta Disaster`
+	description `<destination> is under attack by pirates. Assist the Republic by delivering <commodity> to the planet as soon as possible.`
+	source
+		near "Mebsuta" 2 4
+	destination "Featherweight"
+	cargo "relief supplies" 178
+	to offer
+		random < 20
+		"combat rating" > 200
+	
+	on offer
+		conversation
+			`You are approached by a Navy officer. "Are you Captain <last>?" You nod. "Would you be able to assist us? A sizable gang of pirates has occupied <system> and raided the spaceport. A Navy fleet is on its way to eliminating the pirates, but early reports say that the spaceport of <planet> is in dire need of supplies. The pirates badly damaged any building above three stories and everything in the warehouses was either stolen or destroyed. We need a ship to bring <cargo> to them ASAP, but there are no available Navy ships capable of doing the job. Could you deliver the supplies for us, Captain?"`
+			choice
+				`	"I'd be glad to help, Officer."`
+					accept
+				`	"Sorry, but I'm not travelling in that direction."`
+					decline
+	
+	npc
+		government "Pirate"
+		personality staying harvests plunders
+		system destination
+		fleet "Large Northern Pirates" 3
+	npc
+		government "Republic"
+		personality staying heroic
+		system destination
+		fleet "Large Republic"
+		fleet "Small Republic" 2
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Mirzam"
+		fleet "Large Northern Pirates"
+		fleet "Small Northern Pirates" 3
+	
+	on complete
+		payment
+		payment 7500
+		dialog `The last crate of supplies is loaded off of your ship just as the Navy ships in orbit finish off the pirate fleet. The local authorities thank you for helping, and you are paid <payment>.`
+
+
+
+mission "Southern Blockade"
+	minor
+	name `Spaceport Defenses`
+	description `<stopovers> are under attack and in need of spaceport defenses. Drop off your cargo on these three planets and return to <destination> for payment.`
+	source
+		near "Pherkad" 2 4
+		government "Free Worlds"
+	stopover "Cornucopia"
+	stopover "Solace"
+	stopover "Winter"
+	cargo "spaceport turrets" 75
+	to offer
+		random < 20
+		"combat rating" > 150
+	
+	on offer
+		conversation
+			`A captain in a Free Worlds uniform enters the spaceport and turns on a megaphone. "Attention! The Free Worlds are in need of a captain able to carry <cargo> to <stopovers> for fortifying the spaceports nearest to the pirate system of Men. Free Worlds ships are currently preoccupied with fighting off a pirate incursion into those systems, and we are unable to spare enough ships for the job. Our hope is that these defenses will deter any future incursions. Payment will be <payment> for the first captain who accepts." The spaceport falls quiet as captains contemplate this offer.`
```

Comment:
"The Free Worlds are in need of a captain..."

Just one? Might be better written as "The Free Worlds are in need of volunteers..."